includes/class-sensei-course.php

```diff
@@ -265,9 +265,18 @@ public function showcase_courses_screen() {
 						<span class="sensei-showcase-upsell__price-period"><?php esc_html_e( 'per year, 1 site', 'sensei-lms' ); ?></span>
 					</div>
 
+					<?php
+					$upsell_args = array_merge(
+						Sensei_Pro_Upsell::get_default_utm_params(),
+						[ 'utm_campaign' => 'showcase' ]
```

Comment:
Shouldn't it be in the `get_default_utm_params` argument? And the same for the other cases where we have this.